src/deltacode/__init__.py

```diff
@@ -283,9 +283,17 @@ class Delta(object):
     def __init__(self, score=0, new_file=None, old_file=None):
         self.new_file = new_file if new_file else None
         self.old_file = old_file if old_file else None
+        self.has_license = False
+        self.has_copyright = False
         self.factors = []
         self.score = score
         self.status = ''
+    
+    def add_license(self) :
+        self.has_license = True
+
+    def add_copyright(self):
+        self.has_copyright = True    
```

Comment:
Do we really need separate functions to simply change a field value?

Reply:
@majurg ,no actually it can be modified a bit and made to change the values in place,But I thaught creating a function for it wold be a better approach to it.

Reply:
@majurg  Should I change it?

Reply:
Yes. if all the function does is change a value, there is no need for it. We can add a function later if there is more complex operations to be done. 

Also I question the need for these field anyway. Is there an issue regarding this or...?

Reply:
Yes I think the issue of #109 is meant for it i.g

Reply:
Ok then these fields are fine.